README.md

```diff
@@ -1785,6 +1785,7 @@ Most of these are paid services, some have free tiers.
 * [Reliant](https://github.com/appfoundry/Reliant) - Nonintrusive Objective-C dependency injection.
 * [Kraken](https://github.com/sabirvirtuoso/Kraken) - A Dependency Injection Container for Swift with easy-to-use syntax.
 * [Cleanse](https://github.com/square/Cleanse) - Lightweight Swift Dependency Injection Framework by Square. :large_orange_diamond:
+* [Typhoon](https://github.com/appsquickly/Typhoon) - Poweful dependency injection for iOS & OSX (Objective-C & Swift).
```

Comment:
1. Typo here, should say `Powerful`
2. Remove `for iOS & OSX`, iOS is assumed

Thx